examples/react/src/containers/HistoryMesageList.js

```diff
@@ -24,14 +24,14 @@ export default class HistoryMsgsList extends Component {
           <div>
             {/*// end::HMSG-3.1[]*/}
             {/*// tag::HMSG-4[]*/}
-            {historyMsgs.map( (m, index) =>
-              <li className={styleForMessageSender(m.entry.senderId)} key={m.timetoken}>
-                <div className='msgSentDay'>{getDate(m.timetoken, 'historyMsg', index)}</div>
+            {historyMessage.map( (m, index) =>
```

Comment:
I think `historyMessage` should be plural in this case.  `historyMessages`.

Proper english pluralization of two possible variable names, which is confusing, I know:
historyMesageList. (Message is singular because it indicates the one type of thing in the list.  The List suffix indicates multiple messages of that type.)
historyMessages (Is plural because it indicates that there are many messages stored in the variable.)